lib/base-chips/field-types/text.js

```diff
@@ -4,12 +4,10 @@ const escape = require("escape-html");
 
 module.exports = {
 	name: "text",
-	full_text_search_enabled: function(params){
-		if (params && params.include_in_search){
-			return true;
-		} else if (params && params.max_length > 200){
-			return true;
-		} else {
+	has_index: function(params){
```

Comment:
In a nutshell, 'index' in a database is a data structure that speeds up certain queries. Without indexing, when you want to find all items matching a specified query, the db engine would have to look through each entry in the collection. With indexes, some additional metadata is stored to speed up the process, so the database knows for example where *not* to look :) Both SQL and noSQL databases have some sort of support of indexes: 

https://docs.mongodb.com/manual/indexes/
https://www.postgresql.org/docs/current/static/sql-createindex.html